doc/source/train/user_guide.rst

```diff
@@ -977,6 +977,52 @@ the disk that from which your script was executed from.
     # View the PyTorch Profiler traces.
     $ open http://localhost:6006/#pytorch_profiler
 
+Automatic Mixed Precision
```

Comment:
could you also just add a `.. _torch-amp:` bookmark above this section?

This would add a reference to the section, so if needed we can easily reference this section in other places in the docs.

Reply:
Fixed